openhands/llm/llm.py

```diff
@@ -235,12 +239,14 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
             mock_fncall_tools = None
             # if the agent or caller has defined tools, and we mock via prompting, convert the messages
             if mock_function_calling and 'tools' in kwargs:
+                add_in_context_learning_example = True
+                if 'openhands-lm' in self.config.model or 'devstral' in self.config.model:
+                    add_in_context_learning_example = False
+
```

Comment:
well, i accidentally commited this together.. but probably it is not hurting getting them in :)